### PR TITLE
Fix improper RSA key conversion from `ssh_key` crate

### PR DESCRIPTION
**Pull request by EpicEric**
`rsa` 0.9.7 introduces an error when using a key to sign (likely due to overeager validation during conversion). You can see the changes here: https://diff.rs/rsa/0.9.6/0.9.7/

This commit fixes the version back to 0.9.6 as a quick fix, although this may not be the desired solution.

### CODE REVIEW

**Comment by EpicEric** [comment]
Tracking issue: https://github.com/RustCrypto/RSA/issues/463

**Comment by EpicEric** [comment]
I've updated this with a proper workaround while https://github.com/RustCrypto/SSH/pull/318 is not merged.

**Comment by Eugeny** [comment]
Thank you for investigating this!

File: Cargo.toml [hunk 1]
```diff
@@ -26,7 +26,7 @@ futures = "0.3"
 hmac = "0.12"
 log = "0.4"
 rand = "0.8"
-rsa = "0.9"
+rsa = "=0.9.6"
```
Comment (dignifiedquire) [inline Cargo.toml:29]: I would recommend not pinning to `0.9.6` to make sure this issue doesn't occur in other code paths, given that you have a workaround now

**Pull request merged**